src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,130 @@
-import React from 'react';
-import './FindMovie.scss';
+import React,
+{
+  FC,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
 
+import './FindMovie.scss';
+import cn from 'classnames';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+interface Props {
+  addMovie(movie: Movie): void;
+}
+
+export const FindMovie: FC<Props> = ({ addMovie }) => {
+  const URL = 'https://www.omdbapi.com/?apikey=574c753d&t=';
+  const IMDB_URL = 'https://www.imdb.com/title/';
```

Comment:
in my opinion it is better to keep the constants in a separate file and import where necessary